add jacoco report

annaprookhorova opened the pull request.

annashm1017 committed:
add jacoco report

mkouznetsov-git commented on src/main/java/com/example/CatFamily.java:
```diff
@@ -0,0 +1,8 @@
+package com.example;
+import java.util.List;
+
+public interface CatFamily {
```
Этот интерфейс не нужен для изоляции Lion от Feline.

mkouznetsov-git commented on src/test/java/com/example/LionTest.java:
```diff
@@ -0,0 +1,45 @@
+    @Test
+    public void doesHaveMane() throws Exception {
+        Lion lion = new Lion("Самец");
+        assertEquals(true, lion.doesHaveMane());
```
Здесь лучше сразу использовать assertTrue()

annaprookhorova committed:
update after review
p.s
johnapollon - имя моего компутера, я забыла задать для git пользователя по умолчанию. Причем в github отображается не johnapollon а моя старая учетка от другого курса... Запутанный клубок распутан, жду миллион :)

mkouznetsov-git commented on src/main/java/com/example/Lion.java:
```diff
@@ -6,17 +6,22 @@ public class Lion {
 
+    private Feline feline;
+
     public Lion(String sex) throws Exception {
```
Этот конструктор строит неполноценный объект. У тебя есть новый конструктор с двумя параметрами, теперь этот уже не нужен.
